helloworld.c

```diff
@@ -1,7 +1,8 @@
-#include <stdio.h>
+#include <stdio.h>/*---------decclaring library---------*/
 
-int main(void)
+int main()/*-------------decalaring as a main function of program----------*/
 {
+    /*-------------printing hello world-----------*/
     printf("Hello World!")
-    
+    return 0 ; 
```

Comment:
Thank you for your contribution! I am sorry for the delay in reviewing but this "return 0;" is not necessary. Could you please remove this? Thank you again.